includes/admin/views/builder/settings-fields.php

```diff
@@ -90,29 +98,37 @@
 							</div>
 						<# } #>
 
-					<# } else if ( data.is_default_field( field.type ) ) { #>
+					<# } else if ( data.is_default_field( field.type ) || 'switch-multi-input' === field.type ) { #>
+
+						<# field.inputs = data.is_default_field( field.type ) ? [ field ] : field.inputs; #>
 
 						<# if ( -1 === field.type.indexOf( 'switch-' ) || ( -1 !== field.type.indexOf( 'switch-' ) && data.is_switch_condition_met( field ) ) ) { #>
 							<div class="llms-editable-input{{{ field.classes }}}">
-								<input
-									class="llms-input standard"
-									data-attribute="{{{ field.attribute }}}"
-									data-original-content="{{{ data.model.get( field.attribute ) }}}"
-									<# if ( 'datepicker' === field.type ) { #>
-										<# if ( field.datepicker ) { #> data-date-datepicker="{{{ field.datepicker }}}" <# } #>
-										<# if ( field.timepicker ) { #> data-date-timepicker="{{{ field.timepicker }}}" <# } #>
-										<# if ( field.date_format ) { #> data-date-format="{{{ field.date_format }}}" <# } #>
-									<# } #>
-									<# if ( field.hasOwnProperty( 'min' ) ) { #> min="{{{ field.min }}}" <# } #>
-									<# if ( field.hasOwnProperty( 'max' ) ) { #> max="{{{ field.max }}}" <# } #>
-									name="{{{ field.attribute }}}"
-									<# if ( field.placeholder ) { #> placeholder="{{{ field.placeholder }}}" <# } #>
-									type="{{{ field.input_type }}}"
-									value="{{{ data.model.get( field.attribute ) }}}"
-								>
+								<# field.inputs.forEach( input => { #>
+									<div class="llms-input-wrapper{{{ input.col ? ' llms-col llms-col-' + input.col : '' }}}">
```

Comment:
do we really need this additional `col` attribute? can't we just handle all this with CSS only?

Reply:
oh yeah, gonna handle it by passing the class to `field.classes`.